demo/scss/examples.scss

```diff
@@ -16,15 +16,44 @@
 .load-bar-container {
   display: flex;
   align-items: center;
-  padding-left: var(--size-3);
+  padding: 0 var(--size-3);
   background-color: var(--color-8);
   border-radius: var(--radius-2);
 }
 
-.load-bar-container i {
-  color: var(--color-5);
-}
-
 #load-bar {
   width: 90%;
 }
+
+#load-bar-settings {
```

Comment:
Could you use classes instead of ids pls?

Reply:
Sure ! I've done the same for every file, no more styling with ids :)